Cards/MadGraph5_aMCatNLO/DY/DYtoLL_0J_madgraphMLM-pythia8/DYtoLL_0J_madgraphMLM-pythia8.json

```diff
@@ -0,0 +1,21 @@
+{
+    "scheme": "massZ.dat",
+
+    "run_card":{
+        "parton_shower": "pythia8",
+        "ickkw": "1",
+        "xqcut": "10",
+        "maxjetflavor": "5"
+    },
+
+    "user":[
+        "set run_card mmll 50",
+        "set run_card auto_ptj_mjj True"
+    ],
+
+    "fragment": [
```

Comment:
Should be change to something like: 
fragment": ["Generator/ExternalLHEProducer.dat","PartonShower/PowhegEmissionVetoPythia8.dat"],
    "fragment_user": [],
    "fragment_vars": {
        "processParameters": [
  ...
  no ?

Reply:
closing this, it's a bit outdated